core/src/main/java/bio/terra/pearl/core/service/export/EnrolleeImportService.java

```diff
@@ -193,43 +203,68 @@ public List<Map<String, String>> generateImportMaps(InputStream in, ImportFileFo
     }
 
     public Enrollee importEnrollee(String portalShortcode, String studyShortcode, StudyEnvironment studyEnv, Map<String, String> enrolleeMap, ExportOptions exportOptions) {
+        /** while importing handle update for existing import
+         if same enrolle: update enrollee
+         if same participant & same portal.. new enrollee & same profile
+         if same participant & different portal.. new enrollee & new profile
+         **/
 
         DataAuditInfo auditInfo = DataAuditInfo.builder().systemProcess(
                 DataAuditInfo.systemProcessName(getClass(), "importEnrollee")
         ).build();
 
-        /** first create the participant user */
-        ParticipantUserFormatter participantUserFormatter = new ParticipantUserFormatter(exportOptions);
-        ParticipantUser participantUser = participantUserFormatter.fromStringMap(studyEnv.getId(), enrolleeMap);
-        if (participantUser.getUsername() == null) {
-            throw new IllegalArgumentException("username must be provided for enrollee import");
-        }
-        RegistrationService.RegistrationResult regResult = registrationService.register(portalShortcode, studyEnv.getEnvironmentName(), participantUser.getUsername(), null, null);
-        /** temporarily update the profile to no emails since they'll receive a special welcome email */
-        regResult.profile().setDoNotEmail(true);
-        profileService.update(regResult.profile(), auditInfo);
-
-        /** now create the enrollee */
-        EnrolleeFormatter enrolleeFormatter = new EnrolleeFormatter(exportOptions);
-        Enrollee enrollee = enrolleeFormatter.fromStringMap(studyEnv.getId(), enrolleeMap);
-        HubResponse<Enrollee> response = enrollmentService.enroll(regResult.portalParticipantUser(), studyEnv.getEnvironmentName(), studyShortcode, regResult.participantUser(), regResult.portalParticipantUser(), null, enrollee.isSubject());
-        //update createdAt
-        if (enrollee.getCreatedAt() != null) {
-            timeShiftPopulateDao.changeEnrolleeCreationTime(response.getEnrollee().getId(), enrollee.getCreatedAt());
+        //check if participant already exists.. if so use the existing participant
```

Comment:
Haven't tested, but I think you want the logic to look like the below (but please pull appropriate helper methods out so the method isn't so long):
```
public Enrollee importEnrollee(String portalShortcode, String studyShortcode, StudyEnvironment studyEnv, Map<String, String> enrolleeMap, ExportOptions exportOptions) {
        /** while importing handle update for existing import
         if same enrolle: update enrollee
         if same participant & same portal.. new enrollee & same profile
         if same participant & different portal.. new enrollee & new profile
         **/

        DataAuditInfo auditInfo = DataAuditInfo.builder().systemProcess(
                DataAuditInfo.systemProcessName(getClass(), "importEnrollee")
        ).build();

        //
        ParticipantUserFormatter participantUserFormatter = new ParticipantUserFormatter(exportOptions);
        final ParticipantUser participantUserInfo = participantUserFormatter.fromStringMap(studyEnv.getId(), enrolleeMap);
        
        final RegistrationService.RegistrationResult regResult = portalParticipantUserService.findOne(participantUserInfo.getUsername(), portalShortcode, studyEnv.getEnvironmentName())
                .map((ppUser) -> new RegistrationService.RegistrationResult(
                        participantUserService.findOne(participantUserInfo.getUsername(),
                                studyEnv.getEnvironmentName()).orElseThrow(() -> new IllegalStateException("Participant User could not be found or for PPUser")),
                        ppUser,
                        profileService.find(ppUser.getProfileId()).orElseThrow(IllegalStateException::new)
                )).orElseGet(() ->
            registrationService.register(portalShortcode, studyEnv.getEnvironmentName(), participantUserInfo.getUsername(), null, null)
        );

        Enrollee enrollee = enrolleeService.findByParticipantUserIdAndStudyEnvId(regResult.participantUser().getId(), studyEnv.getId()).orElseGet(() -> {
            /** user is not enrolled in this study, so we need to create a new enrollee */
            EnrolleeFormatter enrolleeFormatter = new EnrolleeFormatter(exportOptions);
            Enrollee enrolleeInfo = enrolleeFormatter.fromStringMap(studyEnv.getId(), enrolleeMap);
            /** temporarily update the profile to no emails since they'll receive a special welcome email */
            regResult.profile().setDoNotEmail(true);
            profileService.update(regResult.profile(), auditInfo);

            HubResponse<Enrollee> response = enrollmentService.enroll(regResult.portalParticipantUser(), studyEnv.getEnvironmentName(),
                    studyShortcode, regResult.participantUser(), regResult.portalParticipantUser(), null, enrolleeInfo.isSubject());
            Enrollee newEnrollee = response.getEnrollee();
            //update createdAt
            if (newEnrollee.getCreatedAt() != null) {
                timeShiftPopulateDao.changeEnrolleeCreationTime(response.getEnrollee().getId(), enrolleeInfo.getCreatedAt());
            }if (regResult.participantUser().getCreatedAt() != null) {
                timeShiftPopulateDao.changeParticipantAccountCreationTime(response.getEnrollee().getParticipantUserId(), participantUserInfo.getCreatedAt());
            }
            return newEnrollee;
        });
        
        
        /** now update the profile */
        Profile profile = importProfile(enrolleeMap, regResult.profile(), exportOptions, studyEnv, auditInfo);

        importSurveyResponses(portalShortcode, enrolleeMap, exportOptions, studyEnv, regResult.portalParticipantUser(), enrollee, auditInfo);

        /** restore email */
        profile.setDoNotEmail(false);
        profileService.update(profile, auditInfo);
        return enrollee;
    }
    ```